src/common-crypto/CryptoBackend.ts

```diff
@@ -113,6 +115,16 @@ export interface SyncCryptoCallbacks {
      */
     processKeyCounts(oneTimeKeysCounts?: Record<string, number>, unusedFallbackKeys?: string[]): Promise<void>;
 
+    /**
+     * Handle the notification from /sync or /keys/changes that device lists have
+     * been changed.
```

Comment:
currently we only seem to call it from `/sync`. Is there an intention to change this?

Reply:
(looks like the doc-comment on `handleDeviceListChanges` is some hangover of a long-forgotten era)